src/app/visual/toolbar/toolbar-visual.component.html

```diff
@@ -247,3 +247,109 @@
     </sky-toolbar-section>
   </sky-toolbar>
 </div>
+
+<div
+  id="screenshot-toolbar-viewkeeper"
+  style="padding-top: 10px;"
+  [skyViewkeeper]="['.sky-toolbar-container']"
+>
+  <sky-toolbar>
+    <sky-toolbar-item>
+      <button
+        class="sky-btn sky-btn-primary"
+      >
+        <!-- The margin style here is to propery space the icon and button text in modern theme. This is not possible in the toolbar item with pure css today because you can't distinguish pure text nodes in css. -->
+        <sky-icon
+          icon="plus-circle"
+          [ngClass]="{
+            'sky-margin-inline-compact' : theme == 'modern'
```

Comment:
No. I mostly just did this so that our tests followed the best patterns as well. (And I had to reset the baseline screenshots for modern theme anyway because of the other changes)